docs/user/experiments-api.md

```diff
@@ -0,0 +1,41 @@
+# Using the experiments API
+
+The Glean SDK supports tagging all its pings with experiments annotations. The annotations are useful to report that experiments were active at the time the measurement were collected. The annotations are reported in the optional `experiments` entry in the [`ping_info` section](pings/index.md) of all the Glean SDK pings.
```

Comment:
Would it make sense to mention that:

* Experiment IDs and branches don't need to be pre-defined to Glean? (This is different to all other metric types)
* The extra map is also arbitrary (and just string-to-string, not nested)?